code/game/objects/items.dm

```diff
@@ -631,6 +631,42 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 			to_chat(M, "<span class='danger'>You go blind!</span>")
 	return TRUE
 
+///This proc runs when you hit someone with an item campable of performing a knee jerk reaction test, a measure of brain health
+/obj/item/proc/kneejerk(mob/living/carbon/M, mob/living/carbon/user)
+	var/selected_zone = user.zone_selected
+	var/obj/item/bodypart/r_leg = M.get_bodypart(BODY_ZONE_R_LEG)
+	var/obj/item/bodypart/l_leg = M.get_bodypart(BODY_ZONE_L_LEG)
+	var/obj/item/organ/brain/B = M.getorganslot(ORGAN_SLOT_BRAIN)
+	if(ishuman(M))
+		if(selected_zone == BODY_ZONE_R_LEG) //do they have the leg?
+			if(!r_leg)
+				return
+		if(selected_zone == BODY_ZONE_L_LEG)
+			if(!l_leg)
+				return
+		user.do_attack_animation(M)
+		M.visible_message("<span class='warning'>[user] gently taps [M]'s knee with [src].</span>", \
+			"<span class='userdanger'>[user] taps your knee with [src].</span>")
+		playsound(M, 'sound/weapons/punchmiss.ogg', 20, TRUE, -1)
+		if(B) //do they have a brain?
+			var/BH = B.damage
+			if(!BH || BH < BRAIN_DAMAGE_MILD) //a healthy brain produces a normal reaction
+				M.visible_message("<span class='danger'>[M]'s leg kicks out sharply!</span>", \
```

Comment:
Could you add an extra parameter for the blind? This could actually be crazy useful for blind doctors since they can't use the health analyzer